environment.yml

```
@@ -1,13 +1,16 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-bcellmagic-1.0.0
+name: nf-core-bcellmagic-dev
```

This is good - but requires everything on `dev` to be pointing to use `dev` too (which is anyways a good idea...) 
```suggestion
name: nf-core-bcellmagic-1.0.0dev
```